dockerfiles/executor_image/Dockerfile

```diff
@@ -6,48 +6,21 @@ FROM debian@sha256:fac2c0fd33e88dfd3bc88a872cfb78dcb167e74af6162d31724df69e482f8
 
 RUN apt-get update && \
     apt-get install -y \
-    coreutils curl apt-transport-https ca-certificates gnupg-agent software-properties-common fuse && \
+		# Install skopeo and umoci which we use to unpack OCI images when we're not using docker.
+		# Install iproute2 ("ip" command) to configure networking on host.
+    fuse skopeo umoci iproute2 amazon-ecr-credential-helper && \
+    apt-get upgrade -y && \
     rm -rf /var/lib/apt/lists/* && apt-get clean
 
-RUN ARCH=$(dpkg --print-architecture) && \
+RUN apt-get update && \
```

Comment:
nit: I think we could remove this whole RUN step by provisioning `tini` the same way we provision firecracker/jailer etc. (e.g. adding a `container_layer` and downloading the `tini` binary via an `http_file` provisioned by bazel)

Reply:
Cool, I'll see if I can get that working :D